base/interactiveutil.jl

```diff
@@ -222,6 +222,9 @@ code_warntype(f, t::(Type...)) = code_warntype(STDOUT, f, t)
 typesof(args...) = map(a->(isa(a,Type) ? Type{a} : typeof(a)), args)
 
 function gen_call_with_extracted_types(fcn, ex0)
+     if isa(ex0, Symbol)
```

Comment:
odd 5-space indent here... also seems strange to me that this isn't using dispatch?